lib/ueberauth/strategy/cognito.ex

```diff
@@ -232,10 +246,22 @@ defmodule Ueberauth.Strategy.Cognito do
   end
 
   @doc """
-  Currently doesn't return any additional information.
+  Fetches the fields to populate the info section of the `Ueberauth.Auth` struct.
   """
-  def info(_conn) do
-    %Ueberauth.Auth.Info{}
+  def info(conn) do
+    id_token = conn.private[:cognito_id_token]
+    %Ueberauth.Auth.Info{
+      email:       id_token["email"],
+      name:        id_token["cognito:username"],
+      first_name:  id_token["first_name"],
+      last_name:   id_token["last_name"],
+      nickname:    id_token["nickname"],
+      location:    id_token["location"],
+      description: id_token["description"],
+      image:       id_token["image"],
+      phone:       id_token["phone"],
```

Comment:
Per https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims, this should be mapped from `id_token["phone_number"]`